<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoTitleBar</class>
 <widget class="QWidget" name="VideoTitleBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>678</width>
    <height>50</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="deviceNameLabel">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>10</y>
     <width>72</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight: bold;</string>
   </property>
   <property name="text">
    <string>deviceName</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fullScreenBtn">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>10</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>F</string>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="masterBtn">
   <property name="geometry">
    <rect>
     <x>92</x>
     <y>12</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string notr="true">M</string>
   </property>
  </widget>
  <widget class="QPushButton" name="maxBtn">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>12</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Âè£</string>
   </property>
  </widget>
  <widget class="QPushButton" name="minBtn">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>12</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeBtn">
   <property name="geometry">
    <rect>
     <x>624</x>
     <y>12</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QPushButton" name="recoderBtn">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>R</string>
   </property>
  </widget>
  <widget class="QPushButton" name="vhshowBtn">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>12</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>VH</string>
   </property>
  </widget>
  <widget class="QPushButton" name="copyBtn">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>C</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pasteBtn">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>p</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
